src/Fabulous.Core/ViewConverters.fs

```diff
@@ -200,6 +200,18 @@ type CustomContentPage() as self =
         base.OnSizeAllocated(width, height)
         sizeAllocated.Trigger(width, height)
 
+/// A custom SearchHandler which exposes the overridable methods OnQueryChanged, OnQueryConfirmed and OnItemSelected as events
+type CustomSearchHandler() =
```

Comment:
This control would be better placed in Fabulous.CustomControls, but needs ViewElement

Reply:
To fix this, we will need to reorganize the projects so that ViewElement is the same assembly than the custom controls.